### ISSUE TITLE
Specify headers to use regardless of reflection

### ISSUE THREAD

**Issue by rosshadden** [fullstorydev/grpcui#85]
As the docs say, `-reflect-header`s are only used for reflection requests:
> These headers will only be used during reflection requests

Is there a way to pass headers that get used regardless?

**Comment by jhump** [comment]
@rosshadden, `-H` sets headers used for all requests.

**Comment by rosshadden** [comment]
@jhump Is that an upcoming change? The version I have installed (latest from `go get`) doesn't have or support that option, and I can't find it in the source code in `master` either.

**Comment by jhump** [comment]
Oh weird. How did we lose that... I thought it was there all along. So it's an accidental omission. It needs to be added (back?)

**Comment by jhump** [comment]
Oh wait... Sorry, I was looking at `grpcurl -h` when I saw it.

It was in fact intentional that it not be in `grpcui`: because the user specifies the metadata in the UI.

I guess if we did have a `-H` option, it would just be used to populate the default values in the UI. I certainly wouldn't want there to be anything passed to the server that the user couldn't see or control. Maybe it should be called `-default-headers` instead.

The `-reflect-headers` are still needed because they are issued transparently -- the user does not invoke them through the UI but rather they are executed _before_ the UI can be shown to populate the drop-downs and schema info.
